mobile/function/route_list_functions.js

```diff
@@ -0,0 +1,138 @@
+            const results = [];
+
+            data.map((route) => {
+                results.push(
+                    <React.Fragment key={route.id}>
+                        <View style={card.layout}>
+                            <View style={styles.inlineIconText}>
+                                <MaterialCommunityIcons
+                                    name="walk"
+                                    size={30}
+                                    color="black"
+                                />
+                                <Text style={card.routeTitle} numberOfLines={1}>
+                                    {route.name}
+                                </Text>
+                            </View>
+                            <View
+                                style={styles.inlineIconText}
+                                ellipsizeMode="head"
+                            >
+                                <Feather name="info" size={30} color="black" />
+                                <Text
+                                    style={card.routeSubTitle}
+                                    numberOfLines={1}
+                                >
+                                    {route.extra}
+                                </Text>
+                            </View>
+                            <View style={styles.inlineIconText}>
+                                <Entypo
+                                    name="direction"
+                                    size={30}
+                                    color="black"
+                                />
+                                <Text style={card.routeSubTitle}>
+                                    {route.distance}
+                                </Text>
+                            </View>
+                            <View style={card.bottomLayout} />
+                            <Text style={card.routeText} numberOfLines={6}>
+                                {route.description}
+                            </Text>
+                            <View style={styles.inlineIconText}>
+                                <View
+                                    style={{
+                                        width: "100%",
+                                        justifyContent: "space-between",
+                                        flexDirection: "row",
+                                    }}
+                                >
+                                    <Pressable style={card.infoButton}>
+                                        <Ionicons
+                                            name="information"
+                                            size={24}
+                                            color="black"
+                                        />
+                                    </Pressable>
+                                    <Pressable style={card.cardButton}>
+                                        <Text>ZIE ROUTE</Text>
```

Comment:
Hier moet een nav.navigate komen, maar ik weet niet of we die er nu al in willen zetten, of pas als die pagina bestaat.